src/plugins/vis_type_timeseries/server/lib/vis_data/get_table_data.js

```diff
@@ -58,12 +60,17 @@ export async function getTableData(req, panel) {
       indexPatternObject,
       capabilities
     );
-    const [resp] = await searchStrategy.search(req, [
-      {
-        body,
-        index: panelIndexPattern,
-      },
-    ]);
+    const [resp] = await searchStrategy.search(
+      req,
+      [
+        {
+          body,
+          index: panelIndexPattern,
+        },
+      ],
+      {},
```

Comment:
what is the empty object for

Reply:
This is the options, which the existing implementation does not used so here, an `{}` is passed in. The `dataSourceId` needs to be passed as an argument.